posts/2015-06-12-best-of-web-2015.md

```diff
@@ -0,0 +1,58 @@
+La première édition du Best of Web s’est tenue à Paris le vendredi 5 Juin 2015 à la Grande Crypte de Paris.
+L’idée de cette rencontre est de rassembler les meetups parisiens orientés web et permette à ceux qui les ont loupés de découvrir leurs meilleurs talks.
+Bien évidemment, une grosse majorité des talks étaient tournés vers le JS mais pas seulement, on a pu découvrir pas mal de choses.
+L’événement était sur une seule trame et ce n’est pas plus mal car ça permet de se confronter à des sujets qu’on n’aurait pas forcément choisi si le choix des conférences était plus ouvert.
```

Comment:
I would write __cela__ instead of __ça__. It would make the text more academic, less street talking.